### DISCUSSION TITLE
Thumb up/down satisfaction emoji UI on bot framework composer

### DISCUSSION THREAD

**Discussion by AmirZair** [microsoft/BotFramework-Composer#5791] [Q&A] [answered]
Going through the issues page of bot framework composer, i stumbled on this [issue](https://github.com/microsoft/BotFramework-Composer/issues/4773#issuecomment-731111648) which shows an interesting UI card for getting the user's satisfactory : 

![image](https://user-images.githubusercontent.com/24612735/107932302-b0a30180-6f7d-11eb-8ab9-89881c339318.png)

Does this looks like an adaptive card ? can we reproduce that ?

**Comment by dmvtech** [comment, marked as answer]
Hi @AmirZair This should doable. An an adaptive card should work.

Since this isn't really related to Composer, but for adaptive cards; it would be more appropriate for how-to questions to be asked on [Stack Overflow](https://stackoverflow.com/questions/tagged/botframework+adaptive-cards). Please ask there.

**Reply by AmirZair** [reply]
Just [did](https://stackoverflow.com/questions/66240525/thumb-up-down-satisfaction-emoji-ui-on-bot-framework-composer-with-adaptative-ca), thank you
